### DISCUSSION TITLE
pool heat pump not supported??

### DISCUSSION THREAD

**Discussion by csfolmer** [tuya/tuya-home-assistant#933] [Q&A]
Hi got a heat pump with Tuya intergrated.

![PXL_20230619_194610940](https://github.com/tuya/tuya-home-assistant/assets/75448794/4d88189b-1e0e-4d2e-9fe2-df892163f6f3)
But HA keep saying it's a unsupported device :-(
 
For as I know HA will support heat pump of tuya.
In the Tuya IoT it say it's a Device ID 000110028cce4eeafbe2
Firmware of the device: V3.1.4

How can I find it's supported or not... and how can I make it to work?
Would like to request, but what is exact the TUya device. it that the ID??

![PXL_20230619_194630676](https://github.com/tuya/tuya-home-assistant/assets/75448794/e5d63194-f02f-412d-bc4d-8bfdeb89c268)

**Comment by jaskedf** [comment]
I have the same issue with a Mr. Silence full Inverter Heat pump
